showcase/src/e2e-test/app.spec.js

```diff
@@ -0,0 +1,13 @@
+const BASE_URL = "http://bs-local.com:3000";
+
+describe("General tests:", () => {
+	beforeEach(async () => {
+		await browser.driver.get(BASE_URL);
```

Comment:
I'm not a 100% sure how protractor deals with this. But wouldn't the function resolve before there was a response from the get? I think 
`beforeEach(browser.driver.get(BASE_URL))`
might also work. 🤔

Reply:
Well, normally the `async` function should wait until the promise inside resolves. In fact I don't really know which `async` function is used in this test because there is no import for such function like we do in the unit tests where we import the function from Angular. Maybe it is a function from protractor or jasmine?